src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5,6 +5,8 @@ databaseChangeLog:
       file: db/changelog/changeset/init/02-init-problem-tags.yaml
   - include:
       file: db/changelog/changeset/init/03-init-problem-imports.yaml
```

Comment:
What is difference between testImports and imports?

Reply:
Divided imports. testImports intended for test class, imports for source class.

Reply:
Test class is written by admins mostly, so when admin writes it he should make sure that the imports already included to the test class. Do you think separate imports for test are most suitable?

Reply:
Yes, in order to avoid specifying imports separately for each class, a separate table for test imports was implemented. It allows you to make changes once without editing imports in each individual class.